manifests/init.pp

```diff
@@ -191,19 +191,19 @@
   }
 
   $conffile = "${gitolite::userhome}/.gitolite/conf/gitolite.conf"
-  $keydir   = "${gitolite::userhome}/.gitolite/keydir"
+  $keydir   = "${gitolite::userhome}/.gitolite/key_dir"
```

Comment:
theres reference to this variable in admin.pp line 80 and 76, we should not change it's value.

Reply:
So if I understand this well, we keep the `$keydir` var with its original value even if the path seems incorrect, but we keep the changes concerning lines 199 and 206 ?

(i'm a bit tired, sorry if it was obvious)

Reply:
yes correct.
in line 199 and 206 the path is definitily incorrect (gitoline instead of gitolite). But I think the
path in $keydir is correct. (if you disagree please let me know).
I already did a commit like that in dev_cirrax branch (which also corrects the wrong spec tests).

Reply:
Well I haven't used the module since I opened the pull request so I can't tell if there is an issue with the path; it worked with these changes. If you fixed it in 710b66e, feel free to close it :)

Reply:
it probably worked for you because the everything else needed also used the other path.
Important is, if you upgrade, you probably have to move the path manually before you run puppet, or remove the old path after puppet run (since puppet should manage everything in the the path. I will release now a new version with dev_cirrax branch and close this request.

Thanks for pointing to the error !